ntbs-service/DataAccess/NotificationRepository.cs

```diff
@@ -406,15 +406,22 @@ public async Task<IEnumerable<Notification>> GetInactiveNotificationsToCloseAsyn
         public async Task<NotificationGroup> GetNotificationGroupAsync(int notificationId)
         {
             var notification = await _context.Notification.SingleAsync(n => n.NotificationId == notificationId);
+            return notification.GroupId.HasValue
+                ? await GetNotificationGroupAsync(notification.GroupId.Value)
```

Comment:
Is this supposed to be calling the new method GetNotificationGroupByIdAsync?

Reply:
Haha. I'm such an idiot